dashpay/src/main/kotlin/org/dashevo/dashpay/BlockchainIdentity.kt

```diff
@@ -666,8 +703,8 @@ class BlockchainIdentity {
     }
 
     fun saveNewUsername(username: String, status: UsernameStatus) {
-        val salt = saltForUsername(username, false)
-        saveUsername(username, status, salt, true)
+        //val salt = saltForUsername(username, false)
```

Comment:
Any reason why this was commented?

Reply:
I found that saltForUsername(username, false) was being called twice (first here and then during username registration) and the salt value here was never used.  This may be due to copying buggy iOS code or that we don't have a saving mechanism in this library.

Reply:
//TODO is placed on these save methods

Reply:
addressed here:  https://github.com/dashevo/android-dashpay/pull/23/commits/c9a2d3612f9a872a7fb49fe607beabb82283fd06